client/my-sites/hosting/php-version-card/index.js

```diff
@@ -69,6 +69,12 @@ const PhpVersionCard = ( {
 				} ),
 				value: recommendedValue,
 			},
+			{
+				label: translate( '8.0', {
```

Comment:
Do we want it to be experimental for now? Just so that users know it's not fully tested?